pages/cathocomponents.js

```diff
@@ -12,130 +25,174 @@ import {
+        <HeaderSimple gtmPrefix="next" />
+      </div>
+      <div class=" ComponentsContainer  FlexContainer ColoredBackground">
+        <div class="PasswordAndBarChartContainer RoundedEdges WhiteBackground PaddingLeftRight">
+          <h4  class="TextFormattingH4">Password Field</h4>
+          <h6>
+            Password Field is a component used when a password input with
+            validations is needed.
+          </h6>
+          <PasswordField
+            password={{ label: "Password" }}
+            onValidate={(valid) => {
+              console.log("valid", valid);
+            }}
+          />
+        </div>
+        <div class="PasswordAndBarChartContainer RoundedEdges WhiteBackground PaddingLeftRight">
+          <h4  class="TextFormattingH4">BarChart</h4>
+          <h6>
+            A simplebar chart component. Used to show sorted data in columns.
+          </h6>
+          <BarChart data={barCharData} />
+        </div>
+      </div>
+      <div>
+        <h4  class="TextFormattingH4">CommentsBox</h4>
+        <h6>A simple way of displaying a form or alert.</h6>
+        <div class="CommentsBoxContainer ColoredBackground">
+          <CommentsBox>
+            <p>This is a test message for the CommentsBox component.</p>
+          </CommentsBox>
+        </div>
+      </div>
+      <div class="FlexContainer ColoredBackground PaddingTopBotttom ">
+        <div class="WhiteBackground PaddingLeftRight RoundedEdges">
+          <h4  class="TextFormattingH4">JobCard</h4>
+          <JobCard
+            jobID="123"
+            jobURL="https://www.catho.com.br"
+            name="JobCard"
+            company={{
+              name: "This is a JobCard",
+              avatar: undefined,
+              isConfidential: false,
+              stamps: [
+                {
+                  icon: "person",
+                  description: "",
+                },
+              ],
+            }}
+            salaryRange="R$0-R$99999"
+            benefits={[
+              {
+                name: "Alimentação",
+                icon: "receipt",
+              },
+            ]}
+            publishDate={new Date(2023, 5, 25, 13).getTime()}
+            updateDate={new Date(2023, 4, 25, 13).getTime()}
+            locations={[
+              {
+                name: "Remote",
+                quantity: 15,
+                url: "https://www.catho.com.br",
+              },
+            ]}
+            quantityTotal={10}
+            description="Job Card is a component to standardise Catho's job posting."
+            isPCD={false}
+            isExtended={false}
+            isCompatible={false}
+            isFavorite={false}
+            cta={{
+              type: "DEFAULT",
+              dateSent: new Date(2022, 6, 10, 13).getTime(),
+              timeToUmblock: new Date(2022, 6, 10, 14).getTime(),
+              subscribeLink: "https://www.catho.com.br",
+            }}
```

Comment:
How about cleaning up the code even more and passing all properties to the mock object? In a broader view, could you do this for all components, what do you think?